src/components/ParticipantList/ParticipantInList.tsx

```diff
@@ -26,6 +31,19 @@ export const ParticipantInList = ({
         </Text>
       </div>
       <div className={styler('menuIconContainer')}>
+        {!isLocal && (
+          <div>
+            <Button
+              iconOnly
+              variant={'no-fill'}
```

Comment:
could you use variant="no-fill". No need of braces for string. we will a lint rule later. same for other fields

Reply:
done